src/main/java/fi/otavanopisto/kuntaapi/server/index/IndexableEmergency.java

```diff
@@ -0,0 +1,84 @@
+package fi.otavanopisto.kuntaapi.server.index;
+
+import java.time.OffsetDateTime;
+
+public class IndexableEmergency implements Indexable {
+
+  @Field(index = "not_analyzed", store = true, type = "long")
+  private Long orderIndex;
+
+  @Field
+  private String location;
+
+  @Field(index = "not_analyzed", store = true, type = "date")
+  private OffsetDateTime time;
+
+  @Field(index = "not_analyzed", store = true)
+  private String emergencyId;
+
+  @Field(index = "not_analyzed", store = true)
+  private String organizationId;
+
+  @Field
+  private String description;
+
+  @Override
+  public String getId() {
+    return emergencyId;
+  }
+
+  @Override
+  public String getType() {
+    return "emergency";
+  }
+
+  @Override
+  public Long getOrderIndex() {
+    return orderIndex;
+  }
+
+  public String getLocation() {
+    return location;
+  }
+
+  public void setLocation(String location) {
+    this.location = location;
+  }
+
+  public OffsetDateTime getTime() {
+    return time;
+  }
+
+  public void setTime(OffsetDateTime time) {
+    this.time = time;
+  }
+
+  public String getEmergencyId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 26. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)